contrib/objcwrapper/src/objc_model.nit

```diff
@@ -19,6 +19,33 @@ module objc_model
 class ObjcModel
 	# All analyzed classes
 	var classes = new Array[ObjcClass]
+
+	# Is `objc_type` known by this model?
+	fun knows_type(objc_type: Text): Bool
+	do
+		for c in classes do
+			if c.name == objc_type then return true
```

Comment:
I'm afraid of the potential degenerative cases here.

Have you considered adding a `HashSet[String]` to store the names of the classes and replace this loop by a `class_names.has(obj_type)` ?